### PR TITLE
Maps test

### PR DESCRIPTION
**Pull request by pgilliar**
Added a util function to map and tests

### CODE REVIEW

File: app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsActivity.kt [hunk 1]
```diff
@@ -35,70 +30,28 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnMarker
...
-
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+
         binding = ActivityMapsBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        // Obtain the SupportMapFragment and get notified when the map is ready to be used.
+        // Obtain the SupportMapFragmeOnt and get notified when the map is ready to be used.
```
Comment (TheblueMan003) [inline app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsActivity.kt:45]: typo in comment :)

File: app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsActivity.kt [hunk 2]
```diff
@@ -121,9 +74,8 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnMarker
             location -> if(location != null){
                 lastLocation = location
                 val currentLatLng = LatLng(location.latitude, location.longitude)
-                if(baseLocation != null){
-                    map.moveCamera(CameraUpdateFactory.newLatLngZoom(baseLocation!!, 15f))
-                }else map.moveCamera(CameraUpdateFactory.newLatLngZoom(currentLatLng, 15f))
+                if(utils.getStartingCameraPosition() == null)map.moveCamera(CameraUpdateFactory.newLatLngZoom(currentLatLng, 15f))
```
Comment (TheblueMan003) [inline app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsActivity.kt:77]: Magic value 15f can be put in a constant

**Review by TheblueMan003 — APPROVED** [review]
Looks good to me otherwise

**Review by TruddyTheDuddi — APPROVED** [review]
Approved

**Pull request closed without merging**